Dockerfile.rhel

```diff
@@ -1,7 +1,7 @@
 FROM registry.ci.openshift.org/ocp/builder:rhel-9-golang-1.21-openshift-4.16 AS builder
 WORKDIR /go/src/github.com/openshift/openshift-controller-manager
 COPY . .
-RUN make build --warn-undefined-variables
+RUN GO_COMPLIANCE_INFO=0 make build --warn-undefined-variables
```

Comment:
I'd advice against it, even though it sometimes is painful, this should be silenced elsewhere, if I believe.